src/main/java/frc/robot/subsystems/Flywheel.java

```diff
@@ -12,36 +12,21 @@
 
 import frc.robot.commands.*;
 import edu.wpi.first.wpilibj.livewindow.LiveWindow;
-import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
+import com.ctre.phoenix.motorcontrol.ControlMode;
 // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
-import edu.wpi.first.wpilibj.Solenoid;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
 
-// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
 
 /**
  *
  */
-public class Grabber extends SubsystemBase {
-    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
-
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
-
-    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
-    private Solenoid grabberLock;
-
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
-
-    /**
-    *
-    */
-    public Grabber() {
-        // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
-        grabberLock = new Solenoid(0, 0);
-        addChild("Grabber Lock", grabberLock);
-
-        // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
+public class Flywheel extends SubsystemBase {
+private WPI_TalonFX flywheelMotor;
+    public Flywheel() {
+        flywheelMotor = new WPI_TalonFX(5);
```

Comment:
One thing that's missing here is an initialize function. This would run once when the subsystem is first created, so basic information about the motors can be set. Mainly, this is which direction is forward and what control mode to use. 

The control mode tells it to run based on percentage, not something like encoder ticks. `motor.set(1)` could mean run at 100% or turn 1 encoder tick. You can avoid this issue by specifying a control mode, so `motor.set(ControlMode.PercentOutput, 1)`. Once you do this once, it will remember what control mode it's in.

Sometimes the motor will have to be mounted reversed on the robot. We don't have to manually add negatives in our code, so there's a flag we can toggle that will interpret any 1s as -1s. This is just `motor.setInverted(true)`.

You can either add these lines directly in the constructor of the subsystem, or create an `initializeMotors` function and call that from the init, which is what I've done for the drive train.

Reply:
Also, this looks really weird in GitHub, IDK what's up.